src/crsAdapter/BewotecExpertAdapter.js

```diff
@@ -525,24 +525,40 @@ class BewotecExpertAdapter {
      * @param lineNumber
      * @returns {*}
      */
-    getTravellerByLineNumber(travellers = [], lineNumber) {
+    getTravellerByLineNumber(travellers = [], lineNumber , serviceType) {
         let traveller = (travellers[lineNumber - 1] || {})[CONFIG.parserOptions.attrPrefix];
 
         if (!traveller || !traveller.name) {
             return void 0;
         }
-
-        return {
-            gender: Object.entries(CONFIG.crs.gender2SalutationMap).reduce(
-                (reduced, current) => {
-                    reduced[current[1]] = reduced[current[1]] || current[0];
-                    return reduced;
-                },
-                {}
-            )[traveller.salutation],
-            name: traveller.name,
-            age: traveller.age,
-        };
+        switch (serviceType) {
+            case CONFIG.crs.serviceTypes.hotel:
+                const travellerName = traveller.name.split(' ');
+                return {
+                    gender: Object.entries(CONFIG.crs.gender2SalutationMap).reduce(
+                        (reduced, current) => {
+                            reduced[current[1]] = reduced[current[1]] || current[0];
+                            return reduced;
+                        },
+                        {}
+                    )[traveller.salutation],
+                    firstName: travellerName[0],
+                    lastName: travellerName[travellerName.length - 1],
+                    age: traveller.age,
+                };
+            default:
+                return {
+                    gender: Object.entries(CONFIG.crs.gender2SalutationMap).reduce(
+                        (reduced, current) => {
+                            reduced[current[1]] = reduced[current[1]] || current[0];
+                            return reduced;
+                        },
+                        {}
+                    )[traveller.salutation],
+                    name: traveller.name,
+                    age: traveller.age,
+                };
+        }
```

Comment:
travellers shall always have the same structure. either they have always first and last name or not. a separation by travel type is not needed.

is it really needed to split up the name?

Reply:
Hmm, are you sure about it? I thought, that we have spoken about it few days ago and decided, that would be better to keep it separated. It could became a messy, when someone put his last name as a first in 'name' variable. When we keep it separated we left no space for this kind of things.

Reply:
okay, but then it shall be equal for all service types - the different logic by service type is not necessary.